README.md

```diff
@@ -1,15 +1,22 @@
+# Disclaimer
+[V8](https://v8.dev/) is a community supported project. Our team at Red Hat maintains currency of the latest V8 releases to be used by Node.js on select PowerPC and S390x platforms. A list of platforms as well as their build toolchain supported by Node.js can be found here:
```

Comment:
> on select PowerPC and S390x platforms.

Should be just be explicit and say:  on Linux on IBM Power (ppcle64), AIX and Linux on IBM zSystems / LinuxONE (s390x).

Reply:
> A list of platforms as well as their build toolchain supported by Node.js can be found here

The official list of Node.js platforms, along with their respective toolchains, can be found at: